### PR TITLE
Add Gxit slides notes for GAT 2021

### PR DESCRIPTION
**Pull request by eancelet**
I tried not to repeat what is already on the slides. 
Tell me if I need to write more.

### CODE REVIEW

**Comment by eancelet** [comment]
There is a problem during the site build. I need to find what is going on.

File: topics/admin/tutorials/interactive-tools/slides.html [hunk 1]
```diff
@@ -67,13 +98,30 @@
...
+ - The client only ever speaks to NGINX on the Galaxy server, running on the standard HTTPS 443 port.
+ - Based on the elements provided by the URL, NGINX redirects the requests to Galaxy over a unix domain socket as usual.
+ - On the other hand, HTTP requests targetting interactive tools are redirected by NGINX to a proxy, called GIE Proxy, running on port 8000.
+ - At this point, remember, when an docker container starts, it will be assigned a host (server or node) and a random port on that host (in our example, port 32768). 
+ - This information is stored by Galaxy in a sqlite database.
+ - So the GIE proxy checks in this sqlite database to know which node and port the IT container is to be found.
+ - The GIE proxy forwards the HTTP request to Docker on that node and port.
+ - Docker, in turn, forwards it to the application (e.g. Jupyter) on the container’s port, defined in the wrapper.
+
+ - The source for this figure can be found at: https://docs.google.com/presentation/d/1_4PtfM6A4mOxOlgGh6OGWvzFcxD1bdw4CydEWtm5n8k/ .
```
Comment (hexylena) [inline topics/admin/tutorials/interactive-tools/slides.html:124]: Could you move this into the slide itself? I don't want polly to try and speak the URL.

File: topics/admin/tutorials/interactive-tools/slides.html [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
 logo: "GTN"
+video: true
 
 title: "Galaxy Interactive Tools"
 objectives:
   - Learn the differences between Galaxy Interactive Environments and Galaxy Interactive Tools
   - Have an understanding of what Galaxy Interactive Tools are and how they work
 contributors:
   - natefoo
+  - eancelet
```
Comment (hexylena) [inline topics/admin/tutorials/interactive-tools/slides.html:12]: I don't see you in CONTRIBUTORS.yaml, if you add yourself the build error should go away.

Comment (eancelet) [reply topics/admin/tutorials/interactive-tools/slides.html:12]: :+1:

**Review by hexylena — APPROVED** [review]
This looks great, thank you so much for the contribution!

File: topics/admin/tutorials/interactive-tools/slides.html (hunk 1 above)
Comment (eancelet) [reply topics/admin/tutorials/interactive-tools/slides.html]: Done. Tried something with `.reduce70[.footnote[`

Comment (hexylena) [reply topics/admin/tutorials/interactive-tools/slides.html]: That's perfect.

**Pull request merged**